docs/reference/javascript-api-for-office.md

```diff
@@ -18,109 +19,9 @@ You should use application-specific APIs whenever feasible, and use Common APIs
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <a href="/javascript/api/word"><img src="../images/index/logo-word.svg" alt="Word API reference docs" /></a>
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Word API reference</h3>
-                        <p><a href="/javascript/api/word">JavaScript APIs for building Word add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <a href="/javascript/api/powerpoint"><img src="../images/index/logo-powerpoint.svg" alt="PowerPoint API reference docs" /></a>
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>PowerPoint API reference</h3>
-                        <p><a href="/javascript/api/powerpoint">JavaScript APIs for building PowerPoint add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <a href="/javascript/api/onenote"><img src="../images/index/logo-onenote.svg" alt="OneNote API reference docs" /></a>
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>OneNote API reference</h3>
-                        <p><a href="/javascript/api/onenote">JavaScript APIs for building OneNote add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <a href="/javascript/api/office"><img src="../images/index-landing-page/i_code-blocks.svg" alt="reference docs" /></a>
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Common API reference</h3>
-                        <p><a href="/javascript/api/office">JavaScript APIs that can be used by any Office Add-in.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-</ul>
+|                           |                      |                 |
```

Comment:
Consider using columns instead of a table (see [Columns](https://review.learn.microsoft.com/help/platform/markdown-reference?branch=main#columns)). This would fix the column header warning in the build report.